webapp/channels/src/components/analytics/table_chart.tsx

```diff
@@ -34,23 +31,15 @@ const TableChart = ({
                             data.map((item) => (
                                 <tr key={'table-entry-' + item.name}>
                                     <td>
-                                        <OverlayTrigger
-                                            delayShow={Constants.OVERLAY_TIME_DELAY}
+                                        <WithTooltip
+                                            id={'tip-table-entry-' + item.name}
                                             placement='top'
-                                            overlay={(
-                                                <Tooltip id={'tip-table-entry-' + item.name}>
-                                                    {item.tip}
-                                                </Tooltip>
-                                            )}
+                                            title={item.tip}
                                         >
-                                            <time>
-                                                {item.name}
-                                            </time>
-                                        </OverlayTrigger>
-                                    </td>
-                                    <td>
-                                        {item.value}
+                                            <time>{item.name}</time>
```

Comment:
Can you remove <time> tag from here it looks like this will be the name while the value would be the datetime, but for now we can just remove time tag from here since its not correct

Reply:
Yup, You're right. Removed the <time> tag and wrapped item.name in a <span> to ensure WithTooltip receives a single React element child, as React.Children.only requires.